### PR TITLE
DR-1150: Add semantic versioning to UI

### PR DESCRIPTION
**Pull request by myessail**

### CODE REVIEW

**Review by fboulnois — APPROVED** [review]
We'll just need to remove the branch name before the PR gets pushed out.

File: .github/workflows/dev-image-update.yaml [hunk 1]
```diff
@@ -14,16 +15,25 @@ jobs:
...
+        uses: broadinstitute/datarepo-actions@my-ui-semver
+        with:
+          actions_subcommand: "bumper"
+          role_id: ${{ secrets.ROLE_ID }}
+          secret_id: ${{ secrets.SECRET_ID }}
+          version_file_path: package.json
+          version_variable_name: version
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: "Get gcp credentials"
         uses: broadinstitute/datarepo-actions@0.6.0
```
Comment (smark88) [inline .github/workflows/dev-image-update.yaml]:
```suggestion
        uses: broadinstitute/datarepo-actions@0.16.0
```

File: .github/workflows/dev-image-update.yaml [hunk 2]
```diff
@@ -48,14 +58,14 @@ jobs:
       - name: "Check and edit Helm definition for dev"
         uses: broadinstitute/datarepo-actions@0.6.0
```
Comment (smark88) [inline .github/workflows/dev-image-update.yaml]:
```suggestion
        uses: broadinstitute/datarepo-actions@0.16.0
```

File: .github/workflows/dev-image-update.yaml [hunk 3]
```diff
@@ -48,14 +58,14 @@ jobs:
...
         with:
-          actions_subcommand: 'deploytagupdate'
+          actions_subcommand: "deploytagupdate"
           helm_env_prefix: dev
           helm_imagetag_update: ui
       - name: "Check and edit Helm definition for integration"
         uses: broadinstitute/datarepo-actions@0.6.0
```
Comment (smark88) [inline .github/workflows/dev-image-update.yaml]:
```suggestion
        uses: broadinstitute/datarepo-actions@0.16.0
```

File: .github/workflows/dev-image-update.yaml (hunk 1 above)
Comment (smark88) [reply .github/workflows/dev-image-update.yaml]: looks like the action on commit will tag 0.16.0

**Pull request merged**